PowerUp/app/src/main/java/powerup/systers/com/DisplayPointsActivity.java

```diff
@@ -0,0 +1,56 @@
+package powerup.systers.com;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import powerup.systers.com.datamodel.SessionHistory;
+
+
+/**
+ * Created by chhavi on 9/1/16.
+ */
+public class DisplayPointsActivity extends Activity {
+    private TextView currentPoints;
+    private TextView totalPoints;
+    private Button mapButton;
+    private Button nextScenarioButton;
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.display_points);
+        mapButton = (Button)findViewById(R.id.mapButton);
+        nextScenarioButton = (Button)findViewById(R.id.nextScenarioButton);
+        Intent i = getIntent();
+        int currentPointsValue = i.getExtras().getInt("current_scenario_points");
```

Comment:
Store this string as a static string in a keys class to avoid errors. Feel free to create a new keys class as we will be adding other strings like this one there